lib/rspec/matchers.rb

```diff
@@ -352,6 +352,19 @@ def be_a_kind_of(expected)
     alias_method :be_kind_of, :be_a_kind_of
     alias_matcher :a_kind_of,  :be_a_kind_of
 
+    # Passes if actual.ancestors.include?(expected)
+    #
+    # @example
+    #
+    #   expect(Integer).to    be_a_kind_of(Fixnum)
+    #   expect(String).to     be_a_kind_of(Object)
+    #   expect(String).not_to be_a_kind_of(Integer)
+    def be_a_subclass_of(expected)
+      BuiltIn::BeASubclassOf.new(expected)
+    end
+    alias_method :be_subclass_of, :be_a_subclass_of
+    alias_matcher :a_subclass_of,  :be_a_subclass_of
```

Comment:
All the matcher aliases are spec'd in `spec/rspec/matchers/aliases_spec.rb` so please add one for this as well.